### PR TITLE
Add Post Tags block description and icon

### PR DESCRIPTION
**Pull request by MichaelArestad**
The Post Tags block displays the post's tags in a horizontal list.

Description proposal:

 > Insert a list of tags to your document.

#### Screenshot

image

### CODE REVIEW

File: packages/block-library/src/post-tags/index.js [hunk 1]
```diff
@@ -14,5 +15,7 @@ export { metadata, name };
 
 export const settings = {
 	title: __( 'Post Tags' ),
+	description: __( 'Insert a list of tags to your document.' ),
```
Comment (epiqueras) [inline packages/block-library/src/post-tags/index.js:18]: I don't think this adequately describes what the block does.

It displays the list of tags for the current document.

**Comment by kristastevens** [comment]
Hello again! Here are some thoughts on the copy. I'm not sure how much room we have to work with here, but it would be great if the copy we choose helps to educate new users on the purpose of tags to help learn how to use them. 

Is it possible to link to the support document to the word "tags" so that users directly check out more information?  [https://wordpress.com/support/posts/tags/]? 

Here are some copy options. The first two are succinct, the last two are more descriptive. 
* Add tags to identify your post's topics.
* Add tags to identify the topics of your post. 
* What is your post about? Add tags so that readers interested in the same topics can discover your post. 
* Add tags to identify your post's topics. Use both broad and specific terms. For example, "science" "zoology" "tigers."

Let me know if you have any questions. Thanks for the chance to weigh in here!

**Comment by MichaelArestad** [comment]
> Is it possible to link to the support document to the word "tags" so that users directly check out more information? [https://wordpress.com/support/posts/tags/]?

I don't think so. At least that was a limitation in the past. @epiqueras do you know if there can be a link in the description or if there is precedent?

I like something like this:

> Add tags to identify your post's topics. For example, "science" "zoology" "tigers."

**Comment by epiqueras** [comment]
They can contain anything you want. Potentially, even components.

**Comment by shaunandrews** [comment]
This is the same icon as the Tag Cloud widget block:

image

Maybe its not a problem, or maybe we should change the Tag Cloud icon to something else.

--

> Add tags to identify your post's topics.
> Add tags to identify the topics of your post.
> What is your post about? Add tags so that readers interested in the same topics can discover your post.
> Add tags to identify your post's topics. Use both broad and specific terms. For example, "science" "zoology" "tigers."

I think these makes if the block let you edit the tags, but I'm not sure you can — I believe this block displays (and _only_ displays) the existing tags associated with a post.

**Comment by MichaelArestad** [comment]
> Maybe its not a problem, or maybe we should change the Tag Cloud icon to something else.

I think it's probably fine. We use the same icon for site title and headings. We could look into some sort of identifier to visually mark these blocks as special somehow.

> I think these make sense if the block lets you edit the tags — but I'm not sure you can. I believe this block displays (and only displays) the existing tags associated with a post.

At some point adding/editing would be great, especially when creating a post, but right now it doesn't have that functionality.

Perhaps a qualifier could help. Something like:

> Display a post's tags.

**Comment by MichaelArestad** [comment]
Updated with something short and sweet:
image

**Review by epiqueras — APPROVED** [review]

**Pull request merged**